src/components/Tailwind/Panels/FeatureNotSupported.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChainId } from '@halodao/sdk'
+import { NETWORK_LABEL } from 'constants/networks'
+import { useActiveWeb3React } from 'hooks'
+import React from 'react'
+import PageWarning from '../Layout/PageWarning'
+
+interface FeatureNotSupportedProps {
+  isIsolated: boolean
+}
+
+const FeatureNotSupported = ({ isIsolated }: FeatureNotSupportedProps) => {
+  const { chainId } = useActiveWeb3React()
+
+  const caption = 'Ooops! Sorry this feature is not supported in ' + NETWORK_LABEL[chainId as ChainId] + '.'
```

Comment:
Should we use a template?

```suggestion
  const caption = `Ooops! Sorry this feature is not supported in ${NETWORK_LABEL[chainId as ChainId]}.`
```

Reply:
Yep, better!